docs/source/run_model_in_simulation.rst

```diff
@@ -0,0 +1,204 @@
+Run a model in gem5-Aladdin simulation
+======================================
+
+Following the tutorial in :ref:`label_build_python_model`, now we are able to
+create a DL model using the SMAUG Python APIs. In this tutorial, we will
+proceed to introduce the steps of running a model in gem5-Aladdin simulation.
+
+Build gem5-Aladdin
+------------------
+
+First, if you haven't built gem5-Aladdin (cloned in
+:code:`/workspace/gem5-aladdin`), use the following command to build it::
+
+    python2.7 `which scons` build/X86/gem5.opt PROTOCOL=MESI_Two_Level_aladdin -j2
```

Comment:
We really need to figure out how to fix this scons issue so we don't have to do this...

Reply:
Yeah, python2.7 is ugly. I think gem5 now has stable python3 support, I'll take a look.